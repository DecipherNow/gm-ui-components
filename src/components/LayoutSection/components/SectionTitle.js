import styled from "styled-components";
import { keen } from "style/styleVariables";

const SectionTitle = styled.h3`
  color: inherit;
  font-size: ${props => props.theme.FONT_SIZE_BASE};
  font-weight: ${props => props.theme.FONT_WEIGHT_SEMIBOLD};
  margin: 0;
`;

SectionTitle.defaultProps = {
  theme: keen
};

export default SectionTitle;
