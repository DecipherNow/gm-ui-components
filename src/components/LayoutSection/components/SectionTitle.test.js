import React from "react";

import SectionTitle from "./SectionTitle";

describe("SectionTitle", () => {
  it("matches snapshot", () => {
    const aSectionTitle = shallow(<SectionTitle />);
    expect(aSectionTitle).toMatchSnapshot();
  });
});
