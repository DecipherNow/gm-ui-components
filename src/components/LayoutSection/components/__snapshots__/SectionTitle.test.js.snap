// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle matches snapshot 1`] = `
.c0 {
  color: inherit;
  margin: 0;
}

<h3
  className="c0"
/>
`;
